spec/std/dir_spec.cr

```diff
@@ -364,7 +364,7 @@ describe "Dir" do
     end
 
     it "raises" do
-      expect_raises_errno(Errno::ENOENT, "Error while changing directory to '/nope'") do
+      expect_raises_errno(Errno::ENOENT, {{ flag?(:win32) ? /SetCurrentDirectory: .* No such file or directory/ : "Error while changing directory to '/nope'" }}) do
```

Comment:
Do we have to spec the error messages? I'd rather just not

Reply:
IMO it's better in order to avoid false positives where the reason for failure might be something entirely different.